draft-ecahc-moderation.md

```diff
@@ -247,28 +247,25 @@ moderator team in ways that are, ideally, integrated into the various
 participation channels the IETF offers. The moderator team SHALL keep
 the identities of participants requesting moderation confidential.
 
-## Transparency
-
-The moderator team SHALL operate according to a consistent
-set of criteria, processes, and actions. The moderator team SHALL
-independently define and execute their role. They SHALL maintain a
-public set of moderation criteria, processes, actions, and other
-material that allows the community to understand and comment on the
-role of the team. The moderator team SHOULD consider adopting
-moderation criteria, processes, and actions that other technical
-communities have found suitable. The moderator team's criteria and
-processes SHALL be developed with community input, but they are not
-expected to be documented in the RFC series.
-
-Some of these processes may rely on automated mechanisms, such as
-rate-limiting emails to lists or messages to chat channels.
-(The IETF's deliberately low bar to participation makes it easy to
-create throw-away personas for such denial-of-service behavior.)
+## Operations of the Moderator Team and Transparency
 
-{:aside}
-> TODO: This gives the moderator team broad freedom to define
-  processes and actions. Should this document define some boundaries
-  for what the moderator team can do?
+Within the bounds of the policies set within this memo and with the
+approval of the IESG, the moderator team SHALL define any additional
+processes and moderation criteria necessary to execute their role.
+Those processes and criteria SHALL be developed with community input
+and made public, but need not be documented in the RFC series.
+
+The intent of this memo is to provide the widest possible freedom of
+action to the moderator team with a few constraints.  Examples of
+actions that could be taken include:
+
+* Automated rate limiting mechanisms;
+* Review and approval of submissions/messages;
+* A Good Talking (or Emailing) To; or
```

Comment:
I don't understand what this is supposed to say.

Reply:
That last one means something like "scolding". I raised this point to Eliot, and he agreed that it might be too much of an idiom.

Reply:
Resolved by b258396